app/resources/query_form.html.template

```diff
@@ -137,11 +137,11 @@
                 value="{{params.query_location}}">
           {% else %}
             <span class="label">
-              {% trans "Location(optional)" %}:
+              {% trans "Home location" %}:
             </span>
             <input id="query_location" name="query_location"
                 class="long-text-input"
-                value="{{params.query_location}}" placeholder="example: Tokyo">
+                value="{{params.query_location}}" placeholder="{% trans 'Street name, Neighborhood, City...etc' %}">
```

Comment:
I believe the correct English styling would be: "Street name, neighborhood, city, etc."